### PR TITLE
Supplement admin-api-cn.md, add route/service/consumer/upstream/ssl definitions.

### PR DESCRIPTION
**Pull request by Miss-you**
Supplement admin-api-cn.md and other files, supplement route / service / consumer / upstream / ssl and other interface definitions and examples

NOTE: Please read the Contributing.md guidelines before submitting your patch:

https://github.com/apache/incubator-apisix/blob/master/Contributing.md#how-to-add-a-new-feature-or-change-an-existing-one

### Summary

Supplement admin-api-cn.md and other files, supplement route / service / consumer / upstream / ssl and other interface definitions and examples

### Full changelog

* [admin-api-cn.md]

### Issues resolved

Fix #XXX

### CODE REVIEW

File: doc/admin-api-cn.md [hunk 1]
```diff
@@ -71,7 +75,29 @@
 - 除了 `uri`/`uris` 是必选的之外，`plugins`、`upstream`/`upstream_id`、`service_id` 这三类必须选择其中至少一个。
 - 对于同一类参数比如 `uri`与 `uris`，`upstream` 与 `upstream_id`，`host` 与 `hosts`，`remote_addr` 与 `remote_addrs` 等，是不能同时存在，二者只能选择其一。如果同时启用，接口会报错。
 
-示例：
+route 对象 json 配置内容：
+
+```json
```
Comment (membphis) [inline doc/admin-api-cn.md]: I think this style is better.

```shell
{
    "id": "1",                  # id，非必填
    "uri": "/release/a",        # uri 路径
    "uris": ["/a","/b"],        # 一组 uri 路径， uri 与 uris 只需要有一个非空即可
    "methods": ["GET","POST"],  #可以填多个方法
    "host": "aa.com",           # host 域名
    "hosts": ["a.com","b.com"], #一组 host 域名， host 与 hosts 只需要有一个非空即可
    "plugins": {},              # 指定 route 绑定的插件
    "priority": 0,              # apisix 支持多种匹配方式，可能会在一次匹配中同时匹配到多条路由，此时优先级高的优先匹配中
    "desc": "hello world",
    "remote_addr": "127.0.0.1",     # 客户端请求 IP 地址
    "remote_addrs": ["127.0.0.1"],  # 一组客户端请求 IP 地址， remote_addr 与 remote_addrs 只需要有一个非空即可
    "vars": [],                 # 由一个或多个 {var, operator, val} 元素组成的列表
    "upstream_id": "1",         # upstream 对象在 etcd 中的 id ，建议使用此值
    "upstream": {},             # upstream 信息对象，建议尽量不要使用
    "filter_func": "",          # 用户自定义的过滤函数，非必填
}
```

File: doc/admin-api-cn.md [hunk 2]
```diff
@@ -147,3 +173,269 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
...
+|---------|---------|----|-----------|----|
+|plugins  |可选 |Plugin|详见 [Plugin](architecture-design-cn.md#plugin) ||
+|upstream | upstream 或 upstream_id 两个选一个 |Upstream|启用的 Upstream 配置，详见 [Upstream](architecture-design-cn.md#upstream)||
+|upstream_id| upstream 或 upstream_id 两个选一个 |Upstream|启用的 upstream id，详见 [Upstream](architecture-design-cn.md#upstream)||
+|desc     |可选 |辅助   |标识服务名称、使用场景等。||
+
+serivce 对象 json 配置内容：
+
+```json
+{
```
Comment (membphis) [inline doc/admin-api-cn.md]: ditto

File: doc/admin-api-cn.md [hunk 3]
```diff
@@ -147,3 +173,269 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
...
+Server: APISIX web server
+
+{"node":{"value":{"upstream":{"nodes":{"39.97.63.215:80":1},"type":"roundrobin"},"plugins":{"limit-count":{"time_window":60,"count":2,"rejected_code":503,"key":"remote_addr","policy":"local"}}},"createdIndex":60,"key":"\/apisix\/services\/201","modifiedIndex":60},"action":"set"}
+```
+
+> 应答参数
+
+目前是直接返回与 etcd 交互后的结果。
+
+[Back to TOC](#目录)
```
Comment (membphis) [inline doc/admin-api-cn.md]: rename `Back to TOC` to `返回目录`

File: doc/admin-api-cn.md [hunk 4]
```diff
@@ -147,3 +173,269 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
...
+|timeout         |可选|超时时间对象|设置连接、发送消息、接收消息的超时时间||
+|enable_websocket     |可选 |辅助|是否允许启用 websocket 能力||
+|hash_on     |可选 |辅助|该参数作为一致性 hash 的入参||
+|desc     |可选 |辅助|标识服务名称、使用场景等。||
+
+upstream 对象 json 配置内容：
+
+```json
+{
+    "id": "1", # id
```
Comment (membphis) [inline doc/admin-api-cn.md]: ditto

File: doc/admin-api-cn.md [hunk 5]
```diff
@@ -147,3 +173,269 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
...
+|---------|---------|----|-----------|----|
+|cert|必需|公钥|https 证书公钥||
+|key|必需|私钥|https 证书私钥||
+|sni|必需|匹配规则|https 证书SNI||
+
+ssl 对象 json 配置内容：
+
+```json
+{
+    "id": "1", # id
```
Comment (membphis) [inline doc/admin-api-cn.md]: ditto

File: doc/admin-api-cn.md (hunk 1 above)
Comment (Miss-you) [reply doc/admin-api-cn.md]: done

Comment (membphis) [inline doc/admin-api-cn.md]: we should use ```shell here.

File: doc/admin-api-cn.md (hunk 3 above)
Comment (membphis) [reply doc/admin-api-cn.md]: any news?

File: doc/admin-api-cn.md (hunk 1 above)
Comment (membphis) [reply doc/admin-api-cn.md]: image

Comment (Miss-you) [reply doc/admin-api-cn.md]: Why is "```shell" better?
And I have corrected it.

Comment (membphis) [reply doc/admin-api-cn.md]: image

**Pull request merged**